### DISCUSSION TITLE
Allow previewing image files in pull request/commits

### DISCUSSION THREAD

**Discussion by Josh65-2201** [community/community#5074] [Pull Requests] [answered]
Currently looking at images in the commits or pull requests will just say it a binary file and can not be previewed. It should just display the image.
![image](https://user-images.githubusercontent.com/26977075/129270986-a953051b-5bb3-465f-bdd2-41ebbc04a67b.png)

**Comment by Simek** [comment, marked as answer]
@josh-65 To see the rich image diff, you need to press the "file" icon located on the right side of the file header:
<img width="691" alt="Screenshot 2021-08-16 100618" src="https://user-images.githubusercontent.com/719641/129531690-e015f750-2632-4ba1-85f1-18e4c2a1b12a.png">

You can read more about the feature in the GitHub docs:
* https://docs.github.com/en/github/managing-files-in-a-repository/working-with-non-code-files/rendering-and-diffing-images

**Reply by Josh65-2201** [reply]
It should really do it automatically in my opinion but at least its there.
